compiler-rt/lib/scudo/standalone/wrappers_c.cpp

```diff
@@ -37,4 +37,16 @@ scudo::Allocator<scudo::Config, SCUDO_PREFIX(malloc_postinit)> SCUDO_ALLOCATOR;
 
 extern "C" INTERFACE void __scudo_print_stats(void) { Allocator.printStats(); }
 
+extern "C" INTERFACE size_t __scudo_get_info(uint32_t topic, void *buffer,
+                                             size_t size) {
+  switch (topic) {
+  case M_INFO_TOPIC_STATS:
+    return Allocator.getStats(reinterpret_cast<char *>(buffer), size);
+  case M_INFO_TOPIC_FRAGMENTATION:
+    return Allocator.getFragmentationInfo(reinterpret_cast<char *>(buffer),
+                                          size);
+  }
```

Comment:
do you want a default case here and below?

Reply:
I can rewrite with a default case. Is it better?

Reply:
Yes, I think it's better to warn any invalid options than silent it

Reply:
Sure. Now it prints a warning message.